angular/lib/experimental.dart

```diff
@@ -128,3 +131,17 @@ bool isDomRenderDirty() => app_view.domRootRendererIsDirty;
 void resetDomRenderDirty() {
   app_view.domRootRendererIsDirty = false;
 }
+
+class _ThrowingSlowComponentLoader implements SlowComponentLoader {
+  @override
+  Future<ComponentRef<T>> load<T>(Type type, Injector injector) {
+    throw new UnimplementedError('Don\'t use SlowComponentLoader');
```

Comment:
I don't know if you are going to land this in this CL, but I'd like to see this as a constant since it is used in two places, and also a little more helpful error message.

Probably too much sass but something like this would be cool:
'You are using the faster, slimmer, improved angular. Sadly that means we don't support loading a component with SlowComponentLoader. Please migrate this code to use ComponentFactories or switch back to the X to continue using SlowComponentLoader if you must.'